### ISSUE TITLE
Support for Annotation Templating

### ISSUE THREAD

**Issue by dfulgham** [flowforge/helm#155]
https://github.com/flowforge/helm/blob/f0d1a5cf531c76aba316d221b07ddda9bd47ed1a/helm/flowforge/templates/service-ingress.yaml#L18C40-L18C40

RE: https://github.com/flowforge/flowforge-driver-k8s/pull/95

Need to support the annotation replacement for serviceName, instanceURL, instanceHost, and instancePort if they exist. (i.e. {{ serviceName }} )

@hardillb

**Comment by hardillb** [comment]
Might have a way

```
{{- if .Values.ingress.annotations }}
  annotations:
    {{- range $keys, $values := .Values.ingress.annotations }}
    {{ $keys }}: {{ $values | replace "{{ serviceName }}" .Values.forge.entryPoint  }}
    {{- end }}
  {{- end }}
```

**Comment by dfulgham** [comment]
@hardillb Im using this now in our deployment (v 1.10.1 ) working well.  I have to manually change service to https and tls unfortunately for editor nodes, but that will get fixed once the tls PR is merged. https://github.com/flowforge/flowforge-driver-k8s/pull/94

One thing that should be added to the docs, is the requirement for space between the "{{" and the variable name, I made the mistake of trying {{serviceName}}  (no spaces) and that fails.

Also don't know if this is a side effect, but I can no longer add the annotation

nginx.ingress.kubernetes.io/force-ssl-redirect: 'true'

It fails with a go unmarshalling error related to the boolean string and fails to deploy

**Comment by hardillb** [comment]
@dfulgham Can you open a new issue for the new problem and supply a sample `values.yml` file that shows the problem please.
